adapters/ddf-adapter/src/main/java/com/connexta/replication/adapters/ddf/DdfNodeAdapter.java

```diff
@@ -320,15 +320,15 @@ public ResourceResponse readResource(ResourceRequest resourceRequest) {
   @Override
   public boolean createResource(CreateStorageRequest createStorageRequest) {
     List<Resource> resources = createStorageRequest.getResources();
-    DdfRestClient client = ddfRestClientFactory.create(hostUrl.toString());
+    DdfRestClient client = ddfRestClientFactory.createWithSubject(hostUrl.toString());
```

Comment:
Can we just pass the URL object instead of converting to string here and below? That way you won't need to create a new URL object in the createWithSubject.